codegen/method.go

```diff
@@ -131,15 +131,19 @@ type StatusCode struct {
 }
 
 const (
-	antHTTPMethod      = "zanzibar.http.method"
-	antHTTPPath        = "zanzibar.http.path"
-	antHTTPStatus      = "zanzibar.http.status"
-	antHTTPReqDefBoxed = "zanzibar.http.req.def"
-	antHTTPReqHeaders  = "zanzibar.http.reqHeaders"
-	antHTTPResHeaders  = "zanzibar.http.resHeaders"
-	antHTTPRef         = "zanzibar.http.ref"
-	antMeta            = "zanzibar.meta"
-	antHandler         = "zanzibar.handler"
+	antHTTPMethod     = "zanzibar.http.method"
+	antHTTPPath       = "zanzibar.http.path"
+	antHTTPStatus     = "zanzibar.http.status"
+	antHTTPReqHeaders = "zanzibar.http.reqHeaders"
+	antHTTPResHeaders = "zanzibar.http.resHeaders"
+	antHTTPRef        = "zanzibar.http.ref"
+	antMeta           = "zanzibar.meta"
+	antHandler        = "zanzibar.handler"
+
+	// AntHTTPReqDefBoxed annotates a method so that the genereted method takes
+	// generated argument directly instead of a struct that warps the argument.
+	// The annotated method should have one and only one argument.
+	AntHTTPReqDefBoxed = "zanzibar.http.req.def"
```

Comment:
we are supposed to deprecate this :(

Reply:
If deprecating the annotation means the (un)marshalling are always on the wrapper struct, then it's easier, we don't even need to look for the imported thrifts.